docs/audio-features.md

```diff
@@ -4,105 +4,148 @@ layout: default
 
 *Following is a list of supported features with their explanations. Unless stated otherwise, extraction algorithms have been adapted from the [yaafe](http://yaafe.sourceforge.net) library.*
 
+<br>
+<br>
 ## Time-domain features
-
+Better documentation of what each audio feature is, what it represents, and what it should be used for, and what its range is"
 ### RMS
 `rms`
 
-The root mean square of the waveform calculated in the time domain to indicate its loudness. Corresponds to the ‘Energy’ feature in YAAFE, adapted from Loy’s Musimathics [1].
+* _Description_: The root mean square of the waveform, that corresponds to its loudness. Corresponds to the ‘Energy’ feature in YAAFE, adapted from Loy’s Musimathics [1].
+* _What Is It Used For_: Getting a rough idea about the loudness of a signal.
+* _Range_: `0.0 - 1.0` where `0.0` is not loud and `1.0` is very loud.
+
+### ZCR
+`zcr`
+
+* _Description_: The number of times that the signal crosses the zero value in the buffer.
+* _What Is It Used For_: Helps differentiating between percussive and pitched sounds. Percussive sounds will have a random ZCR across buffers, where pitched sounds will return a more constant value.
+* _Range_: 0 - ((buffer size / 2) - 1). In Meyda, the default buffer size (`bufferSize`) is 512 and therefore the default ZCR range is `0 - 255`.
 
 ### Energy
 `energy`
 
-The infinite integral of the squared signal. According to Lathi [2].
+* _Description_: The infinite integral of the squared signal. According to Lathi [2].
+* _What Is It Used For_: This is another indicator to the loudness of the signal.
+* _Range_: 0 - `bufferSize`, where `0.0` is very quiet and `bufferSize` is very loud.
 
-### ZCR
-`zcr`
-
-The number of times that the signal crosses the zero value in the buffer. Corresponds to ZCR in jAudio.
 
+<br>
+<br>
 ## Spectral Features
 
 ### AmplitudeSpectrum
 `amplitudeSpectrum`
 
-Frequency domain amplitude spectrum, a transformation from the complex FFT.
+* _Description_: This is also known as the magnitude spectrum. By calculating the Fast Fourier Transform (FFT), we get the signal represented in the frequency domain. The output is an array, where each index is a frequency bin (i.e. containing information about a range of frequencies) containing a complex value (real and imaginary). The amplitude spectrum takes this complex array and computes the amplitude of each index. The result is the distribution of frequencies in the signal along with their corresponding strength.
```

Comment:
Might be good to link to a resource that talks about time domain vs frequency domain, and explains that all signals signals are composed of a sum of sine waves.